shared/utils/relabel-ids.py

```diff
@@ -206,13 +207,16 @@ def drop_oval_checks_extending_non_existing_checks(ovaltree, indexed_oval_defs):
 
             if referreddefinition is None:
                 # There is no oval satisfying the extend_definition referal
-                defstoremove.append((definition, extdefinitionref))
+                incomplete = True
+                sys.stderr.write("WARNING: OVAL check '%s' extends non-existing '%s' "
+                "OVAL definition, removing check from OVAL definitions.\n"
+                % (definition.get("id"), extdefinitionref))
```

Comment:
Could we please only print this once for every def? It's a very small code change and that's what we were doing before.

Could you please change the message to say "OVAL definition" because that's what it is? OVAL check is ambiguous and confusing. There are OVAL defs, states, objects, ... no checks.